plugins/time-resources/src/components/PriorityEditor.svelte

```diff
@@ -13,59 +13,79 @@
 // limitations under the License.
 -->
 <script lang="ts">
-  import { translate } from '@hcengineering/platform'
-  import { ButtonKind, Dropdown, ListItem, themeStore } from '@hcengineering/ui'
-  import { ToDoPriority } from '@hcengineering/time'
+  import { ButtonBaseKind, Dropdown, ListItem, themeStore } from '@hcengineering/ui'
   import { createEventDispatcher } from 'svelte'
+  import { ToDoPriority } from '@hcengineering/time'
+  import { translate } from '@hcengineering/platform'
+  import { todoPriorities } from '../utils'
+  import Priority from './icons/Priority.svelte'
   import time from '../plugin'
 
   export let value: ToDoPriority = ToDoPriority.NoPriority
-  export let kind: ButtonKind | undefined = 'regular'
+  export let kind: ButtonBaseKind | undefined = 'secondary'
   export let onChange: (value: ToDoPriority) => void = () => {}
 
-  let items: ListItem[] = []
+  const dispatch = createEventDispatcher()
 
-  $: fill($themeStore.language)
+  let items: ListItem[] = []
 
-  async function fill (lang: string) {
+  async function fillItems (lang: string) {
     items = [
```

Comment:
I’m not sure, but maybe we need to somehow highlight the currently selected priority in the popup, we do something similar in the tracker (just a thought)